x-pack/plugins/licensing/public/license_feature.test.ts

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { ILicense } from '../common/types';
+import { setup } from './__fixtures__/setup';
+import { Plugin } from './plugin';
+
+describe('licensing feature', () => {
+  let plugin: Plugin;
+  let license: ILicense;
+
+  afterEach(async () => {
+    await plugin.stop();
+    sessionStorage.clear();
+  });
+
+  describe('valid feature', () => {
+    test('isAvailable', async () => {
+      ({ plugin, license } = await setup());
+
+      const security = license.getFeature('security');
+
+      expect(security.isAvailable).toBe(true);
+    });
+
+    test('isEnabled', async () => {
+      ({ plugin, license } = await setup());
```

Comment:
shouldn't these tests reside in `common` folder?

Reply:
The setup for them is different based on server/public.